code-studio/src/js/levelbuilder.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @file Main entry point for scripts used on all level editing pages.
+ */
+'use strict';
+var _ = require('lodash');
+
+window.levelbuilder = window.levelbuilder || {};
+_.extend(window.levelbuilder, {
+  initializeCodeMirror: require('./initializeCodeMirror')
+});
+
+// TODO: Remove when global `CodeMirror` is no longer required.
+window.CodeMirror = require('codemirror');
+// TODO: Remove when global `marked` is no longer required.
+window.marked = require('marked');
```

Comment:
I figured both of these are short-term churn; remember, at least for these requires we're only talking about levelbuilder pages, so once all levelbuilder uses of these globals are gone we can remove these two lines.